### PR TITLE
Add support for boleto transactions

### PR DESCRIPTION
**Pull request by pmatiello**
This PR adds the necessary fields to the `Payment` model in order to enable the creation of "boleto" transactions.

### CODE REVIEW

**Comment by j133y** [comment]
:ship: :it:

File: spec/payment_spec.rb [hunk 1]
```diff
@@ -51,8 +51,39 @@
     }
   }
 
+  let(:boleto_payment) {
+    {
+      'Address' => 'N/A, 1',
+      'Amount' => 1150,
```
Comment (bvicenzo) [inline spec/payment_spec.rb]: Wouldn't be good to map the ```Amount``` and ```BoletoNumber```?

**Comment by bvicenzo** [comment]
:page_facing_up: :octopus:

**Comment by antoniofilho** [comment]
:clap: :clap: :clap:

File: spec/payment_spec.rb (hunk 1 above)
Comment (pmatiello) [inline spec/payment_spec.rb]: Amount is already mapped. :)

I'm not sure what use we would have for BoletoNumber, though. Do you have anything in mind?

Comment (bvicenzo) [inline spec/payment_spec.rb]: Ok, I just thought it would ve util for us... but ok :)

**Comment by rizidoro** [comment]
:money_with_wings:

**Pull request merged**